openstates/id/bills.py

```diff
@@ -194,22 +178,25 @@ def scrape_bill(self, chamber, session, bill_id, short_title=None):
         bill_tables = html.xpath('//table[contains(@class, "bill-table")]')
         title = bill_tables[1].text_content().strip()
         bill_type = get_bill_type(bill_id)
-        bill = Bill(session, chamber, bill_id, title, type=bill_type)
+        bill = Bill(legislative_session=session, chamber=chamber, identifier=bill_id, title=title,
+                    classification=bill_type)
         bill.add_source(url)
-        bill['subjects'] = self._subjects[bill_id.replace(' ', '')]
+        for subject in self._subjects[bill_id.replace(' ', '')]:
+            bill.add_subject(subject)
 
-        if short_title and bill['title'].lower() != short_title.lower():
-            bill.add_title(short_title)
+        # if short_title and title.lower() != short_title.lower():
+        #     print(short_title)
+        #     bill.add_title(short_title)
```

Comment:
pupa support add_title but their is a bug in pupa2billy conversion as i have mention above ... so for now i have commented this out...